src/Kernel/Protocol.class.st

```diff
@@ -47,6 +84,16 @@ Protocol >> initialize [
 	name := self class defaultName
 ]
 
+{ #category : #testing }
+Protocol >> isEmpty [
+	^ self methodSelectors isEmpty
+]
+
+{ #category : #testing }
+Protocol >> isExtensionProtocol [
+	^ self name first = $*
```

Comment:
maybe add comment or something on added content you plan to remove or change in a foreseeable future

Reply:
All things added here are mostly push down from the abstract superclass that I removed. 
But I'll add some comment yes